res/sections/68-EsLes14032017.tex

```diff
@@ -114,25 +114,158 @@ \chapter{Business Continuity and Disaster Recovery}
   title={Domanda su BCP},
   label={bcdr5}
  ]
- 
- \Question Durante un audit del piano di continuità del business, la scoperta 
+
+ \Question Durante un audit del piano di continuità del business, la scoperta
  più sconcertante sarebbe:
  \begin{enumerate}
-   \item L'albero delle chiamate non è stato controllato in maniera debita 
+   \item L'albero delle chiamate non è stato controllato in maniera debita
    negli ultimi sei mesi
    \item L'analisi sull'impatto del business non è stata aggiornata quest'anno
-   \item Un test del sistema di backup e recupero non è stato eseguito 
+   \item Un test del sistema di backup e recupero non è stato eseguito
    regolarmente
    \item Il sito di backup manca di un UPS
  \end{enumerate}
-  
+
 \end{Exercise}
 
 \begin{Answer} [
   ref={bcdr5},
   number={5}
  ]
- 
+
  \Question Risposta esatta: 3
- 
+
+\end{Answer}
+
+% Altro esercizio
+
+\begin{Exercise} [
+  title={Quiz},
+  label={bcdr6}
+ ]
+
+ \Question Il primo e più importante test per verificare il business
+ continuity plan è:
+ \begin{enumerate}
+   \item \textit{Fully operational test}
+   \item \textit{Preparedness Test}
+   \item \textit{Security Test}
+   \item \textit{Desk-Based paper test}
+ \end{enumerate}
+
+\end{Exercise}
+
+\begin{Answer} [
+  ref={bcdr6},
+  number={6}
+ ]
+
+ \Question Risposta esatta: ?
```

Comment:
Anche secondo me, ma non sono sicuro. 
Un'altra che potrebbe avere senso è il preparedeness test perchè provi le varie parti con persone diverse, facendo in pratica addestramento, però in effetti non è il primo passo e ho letto che dovrebbe essere preceduto da un buon desk-based test

Mentre full operational non puoi farlo direttametne prima degli altri due. 
Io opterei per desk-based ma se riusciamo a chiedere a Di Pietro meglio